ros2caret/verb/check_caret_rclcpp.py

```diff
@@ -130,20 +134,24 @@ def _get_file_paths(dir_path: str) -> List[str]:
     def _has_galactic_tp(obj_path: str) -> bool:
         cmd = (f'nm -D {obj_path} | '
                f'grep -e {" -e ".join(galactic_tp_symbol_names)}')
-        return (subprocess.Popen(cmd,
-                                 stdout=subprocess.PIPE,
-                                 shell=True).communicate()[0]
-                )
+        stdout = (subprocess.Popen(cmd,
+                                   stdout=subprocess.PIPE,
+                                   shell=True).communicate()[0]
+                  )
+
+        return True if stdout else False
```

Comment:
How about the following implementation?
```suggestion
        return bool(stdout)
```

Reply:
@atsushi421 
friendly ping

Reply:
@hsgwa 
Thank you for pointing that out.
Fixed in: Thank you very much for pointing that out. :bow: